frontend_lib/src/component/Timeline/Timeline.jsx

```diff
@@ -64,13 +64,12 @@ export class Timeline extends React.Component {
     this.setState({ showEditCommentPopup: true, newComment: comment })
   }
 
-  handleClickValidateEditComment = async () => {
-    const { props, state } = this
-
+  handleClickValidateEditComment = async (comment) => {
```

Comment:
shouldn't this change be reflected on every apps?

Reply:
yes, that's why I changed on Timeline component

Reply:
indeed